installer/conf/omsagent.d/monitor.conf

```diff
@@ -0,0 +1,10 @@
+<source>
+  type monitor_agent
```

Comment:
it's fluentd's standard input plugin .. fluentd-0.12.14/lib/fluent/plugin/in_monitor_agent.rb